core/templates/dev/head/domain/state_card/StateCardObjectFactorySpec.js

```diff
@@ -26,7 +26,8 @@ describe('State card object factory', function() {
     StateCardObjectFactory = $injector.get('StateCardObjectFactory');
 
     _sampleCard = StateCardObjectFactory.createNewCard(
-      'State 1', {}, '<p>Content</p>', '<interaction></interaction>', false);
+      'State 1', {}, '<p>Content</p>', '<interaction></interaction>', {},
```

Comment:
Why is `{}` being passed for the "interaction" argument? Shouldn't we be passing an actual interaction?

Reply:
Done, created an interaction object,